lib/assets/javascripts/turbograft/response.coffee

```diff
@@ -17,7 +17,17 @@ class TurboGraft.Response
     else
       throw new Error("Error encountered for XHR Response: #{this}")
 
+  redirectedTo: () ->
+    @_redirectedTo = @xhr.getResponseHeader('X-XHR-Redirected-To') unless @_redirectedTo
+    @_redirectedTo
+
+  redirectedToNewUrl: () ->
+    @redirectedTo() && (@redirectedTo() != TurboGraft.Response.currentLocation())
```

Comment:
I don't really like relying on `undefined` for a conditional. That's what `@redirectedTo()` will return if not the response header, right?